source/getting-started/building-for-production.md

```diff
@@ -0,0 +1,341 @@
+
+To use one of the following preprocessors, all you need to do is install the appropriate NPM module.
+The respective files will be picked up and processed automatically.
+
+#### LESS
+
+To enable [LESS](http://lesscss.org/), you'll need to add
+[ember-cli-less](https://github.com/gdub22/ember-cli-less) to
+your NPM modules.
+
+```bash
+ember install ember-cli-less
+```
+
+#### Sass
+
+To enable [Sass](http://sass-lang.com/), you'll need to
+install the [ember-cli-sass](https://github.com/aexmachina/ember-cli-sass) addon
+to your project *(both .scss/.sass are allowed)*.
+
+```bash
+ember install ember-cli-sass
+```
+
+#### Compass
+
+To use [Compass](http://compass-style.org/) with your ember-cli app, install
+[ember-cli-compass-compiler](https://github.com/quaertym/ember-cli-compass-compiler) addon using NPM.
+
+```bash
+ember install ember-cli-compass-compiler
+```
+
+#### Stylus
+
+To enable [Stylus](http://learnboost.github.io/stylus/), you must first add
+[ember-cli-stylus](https://github.com/drewcovi/ember-cli-stylus) to your NPM
+modules:
+
+```bash
+ember install ember-cli-stylus
+```
+
+### CoffeeScript
+
+To enable [CoffeeScript](http://coffeescript.org/), you must
+first add [ember-cli-coffeescript](https://github.com/kimroen/ember-cli-coffeescript) to your
+NPM modules:
+
+```bash
+ember install ember-cli-coffeescript
+```
+
+The modified `package.json` should be checked into source control. CoffeeScript
+can be used in your app's source and in tests, just use the `.coffee` extension
+on any file.
+
+The ES6 module transpiler does not directly support CoffeeScript, but using them
+together is simple. Use the `` ` `` character to escape out to JavaScript from
+your `.coffee` files, and use the ES6 syntax there:
+
+```app/models/post.coffee
+`import Ember from 'ember'`
+`import User from 'appkit/models/user'`
+
+Post = Ember.Object.extend
+  init: (userId) ->
+    @set 'user', User.findById(userId)
+
+`export default Post`
+```
+
+Note that earlier versions of the transpiler had explicit support for
+CoffeeScript, but that support has been removed.
+
+### EmberScript
+
+To enable [EmberScript](http://emberscript.com), you must
+first add [broccoli-ember-script](https://github.com/aradabaugh/broccoli-ember-script) to your
+NPM modules:
+
+```bash
+npm install broccoli-ember-script --save-dev
+```
+
+Note that the ES6 module transpiler is not directly supported with Emberscript, to allow use of ES6 modules use the `` ` `` character to escape raw Javascript similar to the CoffeeScript example above.
+
+### Emblem
+
+For [Emblem](http://emblemjs.com/), run the following commands:
+
+```bash
+ember install ember-cli-emblem-hbs-printer
+```
+
+If you're using the older broccoli-emblem-compiler addon, you need to switch over the above ember-cli-emblem-hbs-printer. The older broccoli-emblem-compiler compiles directly to JS instead of Handlebars and therefore is broken on all newer version of HTMLBars.
+
+### Fingerprinting and CDN URLs
```

Comment:
I think all this config stuff should be broken into logically separate pages and put under the Configuring Ember.js section.